sdks/python/apache_beam/runners/common.py

```diff
@@ -95,9 +98,8 @@ def finish(self):
 
   def process(self, element):
     try:
-      with self.logger.PerThreadLoggingContext(step_name=self.step_name):
```

Comment:
Fair point. PTAL. (Worker side code coming separately, but requires this awkward boundary.)